document-loaders/langchain4j-document-loader-google-cloud-storage/pom.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>dev.langchain4j</groupId>
+        <artifactId>langchain4j-parent</artifactId>
+        <version>0.35.0-SNAPSHOT</version>
+        <relativePath>../../langchain4j-parent/pom.xml</relativePath>
+    </parent>
+
+    <artifactId>langchain4j-document-loader-github</artifactId>
+    <name>LangChain4j :: Document loader :: GitHub</name>
+
+    <dependencies>
+
+        <!-- Google Cloud Storage -->
+
+        <dependency>
+            <groupId>com.google.cloud</groupId>
+            <artifactId>google-cloud-storage</artifactId>
+        </dependency>
+
+        <!-- LangChain4j -->
+
+        <dependency>
+            <groupId>dev.langchain4j</groupId>
+            <artifactId>langchain4j-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>dev.langchain4j</groupId>
+            <artifactId>langchain4j</artifactId>
```

Comment:
Yes, for `TextDocumentParser`

Reply:
The other loader modules also depend on it because of `TextDocumentParser` it seems.

Reply:
Oh, I guess it is fine to remove `TextDocumentParser` as a default and ask user to specify `Parser` explicitly. Otherwise `langchain4j` module has to be imported, and it is heavy

Reply:
> The other loader modules also depend on it because of TextDocumentParser it seems.

Yes, but only for tests